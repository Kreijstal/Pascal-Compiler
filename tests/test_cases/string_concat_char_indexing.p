program string_concat_char_indexing;
var
  Input: string;
  Suffix, Prefix: string;
begin
  Input := '123.45';

  Suffix := '';
  Suffix := Suffix + Input[2];
  Suffix := Suffix + Input[4];
  Suffix := Suffix + Input[6];
  WriteLn('Suffix build: ', Suffix);

  Prefix := 'core';
  Prefix := Input[1] + Prefix;
  Prefix := Input[3] + Prefix;
  WriteLn('Prefix build: ', Prefix);
end.
