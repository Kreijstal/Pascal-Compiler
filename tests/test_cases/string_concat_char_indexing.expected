Suffix build: 2.5
Prefix build: 31core
